README.md

```diff
@@ -92,381 +92,172 @@
 <br clear="all" />
 
 ------------------------------------------------------------------------
+# Highlight:
 
-Ivy is both an ML transpiler and a framework, currently supporting JAX,
-TensorFlow, PyTorch, and Numpy.
+Ivy is an open-source machine learning framework that 
+enables you to: 
 
-Ivy unifies all ML frameworks 💥 enabling you not only to **write code
-that can be used with any of these frameworks as the backend**, but also
-to **convert 🔄 any function, model, or library written in any of them to
-your preferred framework!**
+- 🔥 **Autotunes your model**: for optimal framework, compiler, and hardware selection. 
```

Comment:
if the intro says `enables you to` the verbs should reflect that, i.e. `Autotune` instead of `autotunes` and so on